internal/cmd/run/cypress.go

```diff
@@ -122,7 +124,7 @@ func runCypressInDocker(p cypress.Project, testco testcomposer.Client, rs resto.
 	log.Info().Msg("Running Cypress in Docker")
 	printTestEnv("docker")
 
-	cd, err := docker.NewCypress(p, &testco, &testco, &rs, &rs, createReporters(p.Reporters))
+	cd, err := docker.NewCypress(p, region.None, &testco, &testco, &testco, &rs, &rs, createReporters(p.Reporters))
```

Comment:
Why `region.None`?

Reply:
I was using regio region.Region for building URL, but URL now comes in a proper format and I forgot to delete region variable. I fixed this issue.